packages/smart-contracts/scripts/test-deploy-batch-conversion-deployment.ts

```diff
@@ -0,0 +1,88 @@
+import '@nomiclabs/hardhat-ethers';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { deployOne } from './deploy-one';
+
+import {
+  batchConversionPaymentsArtifact,
+  erc20ConversionProxy,
+  erc20FeeProxyArtifact,
+  ethConversionArtifact,
+  ethereumFeeProxyArtifact,
+} from '../src/lib';
+import { chainlinkConversionPath as chainlinkConvArtifact } from '../src/lib';
+import { CurrencyManager } from '@requestnetwork/currency';
+import { deployAddressChecking } from './utils';
+import { BigNumber } from 'ethers';
+
+export const FAU_USD_RATE = 201; // 2.01
+
+// Deploys, set up the contracts
+export async function deployBatchConversionPayment(
+  args: any,
+  hre: HardhatRuntimeEnvironment,
+): Promise<any> {
+  try {
+    console.log('start BatchConversionPayments');
```

Comment:
```suggestion
    console.log('deploy BatchConversionPayments');
```
? Or maybe this was for debugging.